### PR TITLE
F/match results

### PR DESCRIPTION
**Pull request by Vortex148**
should upload data from match results to database. Should also calculate win/loss ratio

### CODE REVIEW

File: apps/entry/views.py [hunk 1]
```diff
@@ -284,6 +284,7 @@ def get(self, request,):
         # get_team_logos()
         # import_first()
```
Comment (AlexanderDefuria) [inline apps/entry/views.py:285]: Cleanup the comments

File: apps/entry/views.py [hunk 2]
```diff
@@ -294,8 +295,6 @@ def get(self, request,):
 
             "img":"NA"
```
Comment (AlexanderDefuria) [inline apps/entry/views.py:296]: Cleanup the whitespace

File: apps/entry/imports.py [hunk 3]
```diff
@@ -115,6 +123,51 @@ def get_team_logos():
                 currentTeam = image_data["teamNumber"]
                 currentImage = image_data["encodedAvatar"]
             team_image.save()
+def get_match_data():
+    for event in Event.objects.all():
```
Comment (AlexanderDefuria) [inline apps/entry/imports.py]: This needs to be split up into smaller functions. Think about creating a method like
`get_match_data_event(event: Event)`  and then looping over the function call.

File: apps/entry/imports.py [hunk 4]
```diff
@@ -115,6 +123,51 @@ def get_team_logos():
...
+
+        try:
+            base_data = make_request(f'{_baseUrl}matches/{event.FIRST_key}?tournamentLevel=Qualification').json()
+            base_data = base_data['Matches']
+            number_of_matches = len(base_data)
+            for match in range(number_of_matches):
```
Comment (AlexanderDefuria) [inline apps/entry/imports.py]: Try this like we did for the events.
```
base_data = make_request(...)
for match in base_data['Matches']:
   ...
```

File: apps/entry/imports.py [hunk 5]
```diff
@@ -115,6 +123,51 @@ def get_team_logos():
...
+                    winner = "B"
+
+                for team in current_match["teams"]:
+                    team_number = team["teamNumber"]
+                    team_info = Team.objects.get(id=team_number)
+
+                    if team["station"][0] == winner:
+                        team_info.totalMatchesWon += 1
+                        # team_info.totalMatchesWon = 0
+                        # team_info.totalMatchesLost = 0
```
Comment (AlexanderDefuria) [inline apps/entry/imports.py]: Cleanup the `#... = 0`

File: apps/entry/imports.py [hunk 6]
```diff
@@ -115,6 +123,51 @@ def get_team_logos():
                 currentTeam = image_data["teamNumber"]
                 currentImage = image_data["encodedAvatar"]
             team_image.save()
+def get_match_data():
+    for event in Event.objects.all():
+
+        try:
```
Comment (AlexanderDefuria) [inline apps/entry/imports.py]: Is it possible to have this try statement surround less code?

**Review by AlexanderDefuria — CHANGES_REQUESTED** [review]
Looks good overall!!

**Comment by Vortex148** [comment]
Updated to separate events rather than all at once

**Review by AlexanderDefuria — APPROVED** [review]

**Pull request merged**